docs/source/hydsim.rst

```diff
@@ -0,0 +1,41 @@
+HydSim
+------
+
+The Mesh Python SDK includes experimental functionality to run hydro simulations and inflow
+calculations using HydSim on the Mesh server. This functionality is under development, and
+therefore subject to change in future releases. Planned but not yet implemented functionality

```

Comment:
Extreme nitpick: I'm not sure about the comma before `and`. That's why I'd split it to two sentences. Additionally, it would be more concise.

```suggestion
calculations using HydSim on the Mesh server. This functionality is under development.
Therefore, it is subject to change in future releases. Planned but not yet implemented functionality
```

You can safely ignore this 😄